src/libraries/System.Windows.Extensions/ref/System.Windows.Extensions.cs

```diff
@@ -12,15 +12,16 @@ public partial class SoundPlayer : System.ComponentModel.Component, System.Runti
         public SoundPlayer() { }
         public SoundPlayer(System.IO.Stream stream) { }
         protected SoundPlayer(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext context) { }
-        public SoundPlayer(string soundLocation) { }
+        public SoundPlayer(string? soundLocation) { }
         public bool IsLoadCompleted { get { throw null; } }
         public int LoadTimeout { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
```

Comment:
Ditto; this should be removed.  Setting SoundLocation to null will throw an exception.